### DISCUSSION TITLE
There is way to use the Bulk Orders and get Order Api

### DISCUSSION THREAD

**Discussion by SergioNtw** [saleweaver/python-amazon-sp-api#1283] [Feature requests]
There is way to use the Bulk Orders and get Order Api, i don't found the how to send bulk operation or the other problem to resolve the quota exceded. There are some prevision to create it?

**Comment by saleweaver** [comment]
You'd use the reports endpoint usually. 

https://developer-docs.amazon.com/sp-api/docs/report-type-values-order

**Reply by SergioNtw** [reply]
But to work and response from amazon, i need all information of the order and it not give me this information. I need to send this info to do the package sent. Regards.

**Reply by saleweaver** [reply]
GET_FLAT_FILE_ORDER_REPORT_DATA_SHIPPING
